pkgs/applications/graphics/pixelnuke/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv, fetchFromGitHub, libevent, glew, glfw }:
+
+stdenv.mkDerivation {
+  pname = "pixelnuke";
+  version = "unstable-2019-05-19";
+
+  src = fetchFromGitHub {
+    owner = "defnull";
+    repo = "pixelflut";
+    rev = "3458157a242ba1789de7ce308480f4e1cbacc916";
+    sha256 = "03dp0p00chy00njl4w02ahxqiwqpjsrvwg8j4yi4dgckkc3gbh40";
+  };
+
+  buildInputs = [ libevent glew glfw ];
+  sourceRoot = "source/pixelnuke";
```

You could put another newline between these two.

```suggestion
  buildInputs = [ libevent glew glfw ];
  
  sourceRoot = "source/pixelnuke";
```

But it's not really important to me.